browser/ui/brave_shields_data_controller.cc

```diff
@@ -376,6 +384,17 @@ void BraveShieldsDataController::SetIsHTTPSEverywhereEnabled(bool is_enabled) {
   ReloadWebContents();
 }
 
+void BraveShieldsDataController::AllowScriptsOnce(
+    const std::vector<std::string>& origins) {
+  BraveShieldsWebContentsObserver* observer =
+      BraveShieldsWebContentsObserver::FromWebContents(web_contents());
+  if (!observer) {
+    return observer->AllowScriptsOnce(origins);
+  }
+
+  ReloadWebContents();
```

Comment:
Q) Should we do reloading here if there is no `BraveShieldsWebContentsObserver`?
I think it just do reloading w/o any changing.